prow/pod-utils/clone/clone.go

```diff
@@ -168,6 +177,17 @@ func gitTimestampEnvs(timestamp int) []string {
 	}
 }
 
+// gitRevParse returns current commit from HEAD in a git tree
+func (g *gitCtx) gitRevParse() (string, error) {
```

Comment:
This is starting to look like the git client :grimacing:

Reply:
😂 where's that even used?

Reply:
Too many other people touching the git client right now, best stay out :grimacing: